src/components/ContributionTable.vue

```diff
@@ -2,13 +2,14 @@
 import DateString from "./table_data/DateString.vue";
 import ToolData from "./table_data/ToolData.vue";
 import UserProfile from "./table_data/UserProfile.vue";
-import { defineProps } from "vue";
+import { computed, defineProps } from "vue";
```

Comment:
Looks like the `computed` import isn't used, please let's remove it for now.

Reply:
defineProps doesn't need to be imported, either, so the line could be removed entirely.

Reply:
Addressed with [e392603](https://github.com/wikimedia/toolhunt-ui/pull/45/commits/e3926030e84e0d3466a0735fbbf6a92421f8a5f2)